src/rgw/rgw_cr_rados.cc

```diff
@@ -323,6 +323,44 @@ int RGWRadosGetOmapKeysCR::request_complete()
   return r;
 }
 
+RGWRadosGetOmapValsCR::RGWRadosGetOmapValsCR(rgw::sal::RGWRadosStore *_store,
```

Comment:
i don't think `RGWSimpleAsyncCR` is a good fit here, because it has to execute synchronously on the `RGWAsyncRadosProcessor` thread pool. `RGWRadosGetOmapKeysCR`/`ValsCR` only issue a single librados call so it's better to use `aio_operate()` with `RGWAioCompletionNotifier`